### PR TITLE
Updates for Initial Release

### PR DESCRIPTION
**Pull request by robertjahn**

### CODE REVIEW

File: setup/createKubernetesServiceAccounts.sh [hunk 1]
```diff
@@ -57,7 +56,7 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: dynatrace-role
+  name: teleport-role
```
Comment (danielmkn) [inline setup/createKubernetesServiceAccounts.sh]: replace it with 'dynatrace-role'?

File: setup/createKubernetesServiceAccounts.sh [hunk 2]
```diff
@@ -105,11 +105,11 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: dynatrace-crb
+  name: teleport-crb
```
Comment (danielmkn) [inline setup/createKubernetesServiceAccounts.sh]: replace it with 'dynatrace-role'?

File: setup/createKubernetesServiceAccounts.sh [hunk 3]
```diff
@@ -105,11 +105,11 @@ rules:
...
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: dynatrace-role
+  name: teleport-role
```
Comment (danielmkn) [inline setup/createKubernetesServiceAccounts.sh]: replace it with 'dynatrace-role'?

**Review by danielmkn — CHANGES_REQUESTED** [review]

File: setup/createKubernetesServiceAccounts.sh (hunk 1 above)
Comment (robertjahn) [reply setup/createKubernetesServiceAccounts.sh]: I agree - just updated and pushed the change

**Review by danielmkn — APPROVED** [review]

**Pull request merged**